### PR TITLE
K8s cni compatible

### PR DESCRIPTION
**Pull request by YaoZengzeng**
make frakti's cni version compatible with k8s

### CODE REVIEW

**Comment by resouer** [comment]
ref https://github.com/kubernetes/frakti/issues/75

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
 	"GoVersion": "go1.7",
 	"GodepVersion": "v79",
 	"Packages": [
-		"github.com/onsi/ginkgo/ginkgo",
```
Comment (resouer) [inline Godeps/Godeps.json]: add this back

File: pkg/hyper/network.go [hunk 2]
```diff
@@ -18,54 +18,121 @@ package hyper
...
-	// only handle bridge plugin now
-	// TODO: support other plugins in the future
-	for _, iface := range r.Interfaces {
-		if iface.Sandbox != "" {
-			// interface information in net ns
-			ret.IfName = iface.Name
-			ret.Mac = iface.Mac
+	var result *NetworkInfo
+	for _, link := range links {
+		if link.Type() != "veth" {
```
Comment (feiskyer) [inline pkg/hyper/network.go]: nit: add a debug log here

File: pkg/hyper/network.go [hunk 3]
```diff
@@ -18,54 +18,121 @@ package hyper
...
+}
+
+func getBridgeNameByIpCompare(ip *net.IPNet) (string, error) {
+	links, err := netlink.LinkList()
+	if err != nil {
+		return "", err
+	}
+
+	for _, link := range links {
+		if link.Type() != "bridge" {
```
Comment (feiskyer) [inline pkg/hyper/network.go]: same here

**Pull request merged**